language-server/modules/langserver-core/samples/hover/hover.bal

```diff
@@ -0,0 +1,54 @@
+
+
+import ballerina/io;
+
+@Description {value:"Struct for represent person's details"}
+@Field{value:"name: string value name of the person"}
+@Field{value:"id: int value id of the person"}
+@Field{value:"age: int value age of the person"}
+type Person {
+    string name;
+    int id;
+    int age;
+};
+
+@Description{value:"Test function to show current package works"}
+@Param{value:"s: string parameter"}
+@Param{value:"sd: int parameter"}
+@Return{value:"return an int"}
+function test1 (string s, int sd) returns int{
+    int a = 0;
+    return a;
+}
+
+
+
+
```

Comment:
May be we can get rid of this unnecessary new lines.

Reply:
@djkevincr these new lines are specifically added to verify the line calculation logic. @NipunaMarcus might be able to confirm further

Reply:
Let's keep them for now.. ill remove them ...